pkg/udevevent/monitor.go

```diff
@@ -122,3 +164,61 @@ func Monitor() {
 		}
 	}
 }
+
+func Subscribe(eventTypes ...UdevEventType) *Subscription {
+	if len(eventTypes) == 0 {
+		eventTypes = []UdevEventType{EventTypeAdd, EventTypeRemove, EventTypeChange}
+	}
+	subscription := Subscription{
+		targetChannel:   make(chan UdevEvent),
+		subscribedTypes: eventTypes,
+	}
+	subscriptions = append(subscriptions, &subscription)
+	return &subscription
+}
+
+func Unsubscribe(sub *Subscription) error {
```

Comment:
do we really use the Unsubscribe method?

Reply:
currently, we probably won't. but kept it to have a complete API.